homeassistant/components/flickelectric/.translations/en.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
We've renamed our translations directory from `.translations` to `translations`. Please rebase on latest dev branch and rename the directory.